cartography/data/jobs/cleanup/aws_import_cloudwatch_event_rules_cleanup.json

```diff
@@ -0,0 +1,15 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (n:AWSCloudWatchEventRule)<-[:RESOURCE]-(:AWSAccount{id: {AWS_ID}})<-[:OWNER]-(:CloudanixWorkspace{id: {WORKSPACE_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
Instead of `AWSCloudWatchEventRule`, let's call this `AWSEventBridgeRule`. This is part of Event Bridge capability of AWS.

Reference: https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoneventbridge.html